tasks/libs/common/utils.py

```diff
@@ -700,3 +700,13 @@ def is_pr_context(branch, pr_id, test_name):
         print(f"PR not found, skipping check for {test_name}.")
         return False
     return True
+
+
+@contextmanager
+def collapsed_section(section_name):
```

Comment:
Should this detect if we're running in CI? I assume printing these section headers and footers locally will have no effect, and will only add noise to the logs.

Reply:
Correct I'll fix this